src/utils/kubernetes.ts

```diff
@@ -112,16 +115,17 @@ export class Kubernetes implements KubernetesApi {
    }
 
    private async waitManifests(
```

Comment:
nit: now that the input isn't a manifest file anymore would a methodname like waitDeployment be better?

Reply:
I think that's a tricky call. That does better describe exactly what the function does but the intent is to wait for the Manifests to be ready. This function might change in the future so it should be better to keep the current name. They don't need to know implementation details based on the function name, just what the function accomplishes.

Reply:
Swapped it around